crypto/.js/src/main/scala/org/bitcoins/crypto/BCryptoCryptoRuntime.scala

```diff
@@ -27,32 +36,38 @@ trait BCryptoCryptoRuntime extends CryptoRuntime {
   private lazy val ecdsa =
     new ECDSA("SECP256K1", sha256, js.constructorOf[SHA256], null)
 
-  private lazy val randomBytesFunc: Int => ByteVector =
+  private lazy val randomBytesFunc: Int => ByteVector = { int =>
```

Comment:
previously nothing was done with the `int` passed into the function, i assume this is what was intended right @rorp ?